commands/translate.ts

```diff
@@ -37,13 +39,37 @@ const apiURL = process.env.IBM_TRANSLATE_API_URL?.toString() || "";
 // =======================================
 
 function sendApology(message: Message, err: any) {
-  message.channel.send(
-    `:skull_crossbones:  An error occured during translation. :fire:\n ` +
-      `Call 1-800-DEVELOPER if the message below looks bad enough.\n` +
-      "```Error: " +
-      err.toString() +
-      "```"
-  );
+  if (err.code == 404) {
+    if (err.body.search("confidence") != -1) {
+      message.channel.send(
+        "Something is not working, I can figure out what language" +
+          " that message was in. If you know the source language try again using the model command\n" +
+          "```!translate <modelid?> (example en-fr for english to french) ```"
+      );
+    } else if (err.body.search("target") != -1) {
+      message.channel.send(
+        "Something is not working, i either can't figure what language" +
+          " this is or you may not have inputed or it is unsuported language.\n" +
+          "Check the supported language list with the following command.\n" +
+          "```!translate list (returns list of target languages)```"
+      );
+    } else {
+      message.channel.send(
+        "Something is not working, the model you ask for does not seem to work.\n" +
+          "You may want to check out the suportted list of languages with this command:" +
+          "`!translate list`\nMake sur to use the values inside the []. If the language code has capital" +
+          " letters in them ie fr-CA make sure to keep them. ex:'!translate en-fr-CA bonjour"
```

Comment:
Could you put some effort into making these messages a bit shorter, adding more detail from the error, and only catching the "confidence" and "target" cases? 

The benefit of passing the error right to the frontend is the user knows exactly what went wrong, and we have to write far less code.

Telling a user "Something is not working" isn't very useful information.

Reply:
They are three case in which the error 404 is raise. 1 a bad target language. 2 to low of a confidence in the detected language and the third is a wrong modelid. In the http error that is sent, the description in the body will contain different key words depending on what caused the error thus the keyword “target” and “ confident.

Reply:
Also I wrote the message to display to a general user. So imagine someone calls onto the bot but is not sure how it works and makes a mistake in the command, then the bot will guide them into the right direction instead of flashing a big error code. The 404 is not a coding error but rather a general user input error

Reply:
I know you want to provide users with clear errors, but the IBM messages say things like "Could not recognize source language" or "Target language unavailable: Chinese" 

All that said, I'd like to see these messages shortened before merging.